source/faq/faq.rst

```diff
@@ -213,20 +213,27 @@ humidity, and publishes them periodically through MQTT. First, you create a
 device of type MQTT with two attributes (temperature and humidity). Then you
 set your actual device to push the data to dojot.
 
-In order to send data to dojot via MQTT (using iotagent-mosca), there are some
+In order to send data to dojot via MQTT (using Mosca or VerneMQ), there are some
 things to keep in mind:
 
-- The topic should look like ``/<tenant>/<device-id>/attrs`` (for instance:
-  ``/admin/efac/attrs``). Depending on how IoT agent MQTT was started (more
-  strict), the client ID must also be set to "<tenant>:<deviceid>", such as
-  "admin:efac".
+- When using Mosca, the topic should look like ``/<tenant>/<device-id>/attrs`` (e.g.:
+  ``/admin/efac/attrs``). Depending on how IoT agent MQTT was started (more strict), the client ID
+  must also be set to ``<tenant>:<deviceid>``, such as ``admin:efac``.
+
+- When using VerneMQ, the topic should look like ``<tenant>:<device-id>/attrs`` (e.g.:
+  ``admin:efac/attrs``). You must also set the username for the client as ``<tenant>:<device-id>``, such
+  as ``admin:efac``, and it should match the same part in the topic. You can also set the client ID
+  too (not required).
 
 - MQTT payload must be a JSON with each key being an attribute of the dojot
   device, such as:
 
 .. code-block:: javascript
 
-  { "temperature" : 10.5,"pressure" : 770 }
+  { "temperature" : 10.5, "pressure" : 770 }
+
+- VerneMQ does not need any additional configuration to permit insecure MQTT communication, unlike
```

Comment:
I think It should be changed

Reply:
I removed it for now. When we add this feature, it'll be updated.